core/pom.xml

```diff
@@ -160,5 +160,19 @@
             <version>1.5.2 </version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.poi/poi -->
+		<dependency>
```

Comment:
I'd suggest removing this dependency since the one below pulls it in. That will make it easier for users because they'll only need to add a single dependency to their project to use this functionality